### PR TITLE
{lib,tools}[GCCcore/11.3.0,foss/2022a] GitPython v3.1.27, buildingspy v4.0.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/buildingspy/buildingspy-4.0.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('matplotlib', '3.5.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('Jinja2', '3.1.2', {
```
Comment (boegel) [inline easybuild/easyconfigs/b/buildingspy/buildingspy-4.0.0-foss-2022a.eb:23]: This exact same version is already included in `Python` 3.10.4

File: easybuild/easyconfigs/b/buildingspy/buildingspy-4.0.0-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        'checksums': ['31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852'],
+    }),
+    ('pytidylib', '0.3.2', {
+        'modulename': 'tidylib',
+        'checksums': ['22b1c8d75970d8064ff999c2369e98af1d0685417eda4c829a5c9f56764b0af3'],
+    }),
+    ('simplejson', '3.16.0', {
+        'checksums': ['b1f329139ba647a9548aa05fb95d046b4a677643070dc2afc05fa2e975d09ca5'],
+    }),
+    ('six', '1.14.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/b/buildingspy/buildingspy-4.0.0-foss-2022a.eb:33]: Both `simplejson` and `six` are already included with Python 3.10.4, and the versions are more recent than what you're using here.

If `buildingspy` has strict version requirements on these versions, we should just relax those (replace `==` with `>=` in `requirements.txt` for example).

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/3059763fb75e80de389cd1b826a5b9f6 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3144.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e67aa2540c57267edd11237a5fada295 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**